includes/managers/controls.php

```diff
@@ -862,6 +875,27 @@ public function remove_control_from_stack( $stack_id, $control_id ) {
 		return true;
 	}
 
+	/**
+	 * Has Stacks Cache Been Cleared.
+	 * @since 3.13.0
+	 * @access protected
+	 * @return bool True if the CSS requires to clear the controls stack cache, False otherwise.
+	 */
+	public function has_stacks_cache_been_cleared() {
```

Comment:
Protected or public?

Reply:
This function is public, like all the other methods of this Manager.

Reply:
Please fix the PHPDoc

Reply:
PHPDoc fixed